h/static/scripts/util/string.js

```diff
@@ -19,7 +24,45 @@ function unhyphenate(name) {
   }
 }
 
+
+/**
+ * Utility function to normalize a string based on our standard pattern of NFKD
+ *
+ * @param  {String}
+ * @returns {String}
+ */
+function normalize(str){
+
+  // This require is coming from a vendor bundle
```

Comment:
Probably worth adding a brief comment that "unorm" provides a `String.prototype.normalize` polyfill